docs/build_developer_images.md

```diff
@@ -0,0 +1,64 @@
+# OpenShift Router Images
+
+There are situations in which you only need to recreate 
+only the router image instead of all the images.
+
+
+## Build Origin
+
+### Clone the origin repo (or your own fork)
+
+git clone https://github.com/openshift/origin.git
+
+cd origin
+
+**In order to speed up the build time we just build for one platform:**
+
+export OS_ONLY_BUILD_PLATFORMS="linux/amd64"
+
+### Build Openshift origin and router images
+
```

Comment:
Split the origin base images apart from the router (i.e. make two sections).  The router one can be called "build dependent images" and use the router as an example